TestCases/compliance-level-3/0080-feel-getvalue-function/0080-feel-getvalue-function.dmn

```diff
@@ -0,0 +1,130 @@
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003" id="_decision_003">
+        <description>Tests FEEL expression: 'get value({a: "foo"}, "a", "bar")' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get value({a: "foo"}, "a", "bar")'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="string" name="decision_003"/>
+        <literalExpression>
+            <text>get value({a: "foo"}, "a", "bar")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_004" id="_decision_004">
+        <description>Tests FEEL expression: 'get value({a: "foo"}, "a")' and expects result: 'foo'</description>
+        <question>Result of FEEL expression 'get value({a: "foo"}, "a")'?</question>
+        <allowedAnswers>foo</allowedAnswers>
+        <variable typeRef="string" name="decision_004"/>
+        <literalExpression>
+            <text>get value({a: "foo"}, "a")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_005" id="_decision_005">
+        <description>Tests FEEL expression: 'get value("foo", "foo")' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get value("foo", "foo")'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="string" name="decision_005"/>
+        <literalExpression>
+            <text>get value("foo", "foo")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_006" id="_decision_006">
+        <description>Tests FEEL expression: 'get value({a: "foo"}, 123)' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get value({a: "foo"}, 123)'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="string" name="decision_006"/>
+        <literalExpression>
+            <text>get value({a: "foo"}, 123)</text>
+        </literalExpression>
+    </decision>
+
+<!--
+    <decision name="decision_007" id="_decision_007">
+        <description>Tests FEEL expression: 'get value(key:"a", m:{a: "foo"})' and expects result: 'foo'</description>
+        <question>Result of FEEL expression 'get value(key:"a", m:{a: "foo"})'?</question>
+        <allowedAnswers>foo</allowedAnswers>
+        <variable typeRef="string" name="decision_007"/>
+        <literalExpression>
+            <text>get value(key:"a", m:{a: "foo"})</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_008" id="_decision_008">
+        <description>Tests FEEL expression: 'get value(k:"a", m:{a: "foo"})' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get value(k:"a", m:{a: "foo"})'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="string" name="decision_008"/>
+        <literalExpression>
+            <text>get value(k:"a", m:{a: "foo"})</text>
```

Comment:
the parameter names in this test looks wrong, and inconsistent with the previous test. However both are commented out. Overall test file approved from my side.